### ISSUE TITLE
NULL deref on pre_obj in history during live tree view

### ISSUE THREAD

**Issue by bennbollay** [godotengine/godot#12758]
**Operating system or device, Godot version, GPU Model and driver (if graphics related):**
Windows 10 64-bit
Godot 2.1.4 compiled locally

Programmatically adding nodes to a scene, placed a breakpoint in _ready, and then attempted to view the scene graph.

Exception:
```
Exception thrown: read access violation.
**pre_obj** was nullptr. occurred
```

```
>	godot.windows.tools.32.exe!EditorHistory::_add_object(unsigned int p_object, const String & p_property, int p_level_change) Line 93
		if (obj->is_type("ScriptEditorDebuggerInspectedObject")) {
			for (int i = current; i >= 0; i--) {
				Object *pre_obj = ObjectDB::get_instance(get_history_obj(i));
==>				if (pre_obj->is_type("ScriptEditorDebuggerInspectedObject")) {
					if (pre_obj->call("get_remote_object_id") == obj->call("get_remote_object_id")) {
						History &pr = history[i];

 	godot.windows.tools.32.exe!EditorHistory::add_object(unsigned int p_object) Line 131
 	godot.windows.tools.32.exe!EditorNode::push_item(Object * p_object, const String & p_property) Line 1427
 	godot.windows.tools.32.exe!ScriptEditorDebugger::_parse_message(const String & p_msg, const Array & p_data) Line 500
 	godot.windows.tools.32.exe!ScriptEditorDebugger::_notification(int p_what) Line 1103
 	godot.windows.tools.32.exe!ScriptEditorDebugger::_notificationv(int p_notification, bool p_reversed) Line 57
 	godot.windows.tools.32.exe!Object::notification(int p_notification, bool p_reversed) Line 865
 	godot.windows.tools.32.exe!SceneTree::_notify_group_pause(const StringName & p_group, int p_notification) Line 863
 	godot.windows.tools.32.exe!SceneTree::idle(float p_time) Line 512
 	godot.windows.tools.32.exe!Main::iteration() Line 1554
 	godot.windows.tools.32.exe!OS_Windows::run() Line 2226
 	godot.windows.tools.32.exe!widechar_main(int argc, wchar_t * * argv) Line 149
 	godot.windows.tools.32.exe!_main() Line 171
 	godot.windows.tools.32.exe!main(int _argc, char * * _argv) Line 183
 	[External Code]		Annotated Frame
 	[Frames below may be incorrect and/or missing, no symbols loaded for kernel32.dll]		Annotated Frame
```

**Comment by akien-mga** [comment]
CC @Geequlim (don't know if it's related to your changes, but you're the debugger expert in my book now :P).

**Comment by Geequlim** [comment]
Sorry it's my fault.

**Comment by akien-mga** [comment]
Fixed by #12771.
